autospec/build.py

```diff
@@ -218,20 +218,25 @@ def package(filemanager, cleanup=False):
         uniqueext = tarball.name
         cleanup_flag = "--no-cleanup-after"
 
+    if mockconfig:
+        config_option = "-r {}".format(mockconfig)
+    else:
```

Comment:
When will this ever happen? If the user doesn't pass `-m` then `mockconfig` will be set to `"clear"`. I believe `-m` will require an argument with the way you have it set up so I'm not seeing how mockconfig could ever be empty (or `False`, the way you use different types is confusing.)

Reply:
It could never be empty with the current state of the code, true. Would you propose that I don't set a default value for `mockconfig` then?

Reply:
no I just propose you always set
```python
config_option = "-r {}".format(mockconfig)
```

Reply:
I left a few things unsaid in my comment, but this is what I implied in the end. Okay, will fix.